monkey/infection_monkey/requirements_windows.txt

```diff
@@ -5,7 +5,7 @@ cffi
 requests
 odict
 paramiko
-psutil==3.4.2
+psutil
 PyInstaller
 six
```

Comment:
The `six` module ought to be removed from our code, and all of its usages replaced with Python3 comp only. This is true for both the agent and the Island.